ui/app/common/gallery/directives/galleryPane.js

```diff
@@ -133,7 +133,7 @@ angular.module('bahmni.common.gallery')
                             });
                         });
                     }
-                    ngDialog.openConfirm({template: '../common/gallery/views/gallery.html', scope: $scope, closeByEscape: true});
+                    ngDialog.openConfirm({template: '../common/gallery/views/gallery.html', scope: $scope, closeByEscape: true, className: 'ngdialog-theme-default gallery-dialog'});
```

Comment:
can you change the order of the classes - 'gallery-dialog ngdialog-theme-default'

Reply:
How will it make a difference?